packages/ui/src/components/Button/Button.tsx

```diff
@@ -20,3 +25,11 @@ export const LinkButton = forwardRef<HTMLAnchorElement, LinkButtonProps>(
 )
 
 LinkButton.displayName = "LinkButton"
```

Comment:
Please refactor LinkButton to ButtonLink to follow same naming convention with ButtonTransparent